src/test/scala/beam/router/MatsimNetworkToOsmConverter.scala

```diff
@@ -0,0 +1,91 @@
+package beam.router
+
+import java.nio.file.Path
+import java.time.{LocalDateTime, ZonedDateTime, ZoneOffset}
+import java.time.format.DateTimeFormatter
+
+import scala.collection.JavaConverters._
+import scala.xml._
+
+import beam.router.r5.DefaultNetworkCoordinator
+import beam.sim.config.BeamConfig
+import org.matsim.api.core.v01.network.{Network, Link => MatsimLink, Node => MatsimNode}
+
+
+class MatsimNetworkToOsmConverter(beamConfig: BeamConfig) {
+  private val uid = "1"
+  private val user = "beam"
+  private val version = "1"
+  private val changeset = "1"
+    val xmlEncode = "UTF-8"
+
+  lazy val matsimNetwork: Network = {
+    val networkCoordinator = DefaultNetworkCoordinator(beamConfig)
+    networkCoordinator.init()
+    networkCoordinator.network
+  }
+
+  def buildXmlFile(outputFile: Path): Path = {
```

Comment:
Should everything except this be private? Also, should there be a documented usecase and/or test to show usage better?

Reply:
I agree. I am adding your suggestions

Reply:
this file was removed. This task is to convert from `Matsim physsym XML` to `OSM XML`